index.html

```diff
@@ -13,7 +15,72 @@
   <![endif]-->
 </head>
 
-<body>
-<!-- Use your own code or past solution for Great Idea Web Page HTML here! -->
-</body>
+  <body>
+
+    <div class="container">
+      <header>
+        <nav>
+          <a href="services.html">Services</a>
+          <a href="#">Product</a>
```

Comment:
# in the dead links, nice. This will make it so that your page doesn't refresh when you click on the dead link